nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -319,7 +317,7 @@ def connect_thread_func(self):
 
             # The traceback usually contains 3 lines, so collect the first
             # three lines then display it.
-            if got_error == 2:
+            if got_error:
```

Comment:
Do we need to close something else here beyond SSH? I get an error from the device side and the sim continues to run without receiving UDP packets:

```
[                                  Simulating... 0%                                   ] ETA: Unknown[INFO] <10.162.177.99> 
[                                  Simulating... 0%                                   ] ETA: Unknown[INFO] No packet received for t=0.001000s.
[                                  Simulating... 0%                                   ] ETA: 6:04:40[ERROR] <10.162.177.99> ENCOUNTERED ERROR!
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/home/ben/anaconda3/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/ben/anaconda3/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ben/Git/nengo-fpga/nengo_fpga/networks/fpga_pes_ensemble_network.py", line 324, in connect_thread_func
    % (remote_ip, '\n'.join(error_strs)))
RuntimeError: Received the following error on the remote side <10.162.177.99>:
Traceback (most recent call last):
  File "/opt/nengo-pynq/nengo_pynq/single_pes_net.py", line 79, in <module>
    fpga_driver = PESDriver(nengo_data)
  File "/opt/nengo-pynq/nengo_pynq/pes_driver.py", line 242, in __init__

[INFO] No packet received for t=0.002000s.
[                                  Simulating... 0%                                   ] ETA: 3:05:36[INFO] No packet received for t=0.003000s.
[                                  Simulating... 0%                                   ] ETA: 2:04:48[INFO] No packet received for t=0.004000s.
```

Reply:
I just realized this is because I didn't put the bitstreams on the device, but in anycase, it should recognize that and not continue running.